package/CHANGELOG

```diff
@@ -156,6 +156,7 @@ Changes
     `__call__` (Issue #2860, PR #2859)
   * deprecated ``analysis.helanal`` module has been removed in favour of
     ``analysis.helix_analysis`` (PR #2929)
+  * Move water bridge analysis from hbonds to hydrogenbonds (Issue #2739 PR #2913)
```

Comment:
Technically new changes go to the top of the list, but I'm willing to let that go.

Reply:
I appreciate your attention to detail.

I merged the whole thing nevertheless, as this has been sitting for way too long.